TaskHouseApi/Repositories/WorkerRepository.cs

```diff
@@ -0,0 +1,76 @@
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TaskHouseApi.DatabaseContext;
+using TaskHouseApi.Model;
+using TaskHouseApi.Repositories;
+
+namespace TaskHouseApi.Repositories
+{
+    public class WorkerRepository : IWorkerRepository
+    {
+        private PostgresContext db;
+
+        public WorkerRepository(PostgresContext db){
+            this.db=db;
+        }
+
+        public async Task<Worker> Create(Worker w)
+        {
+            await db.Workers.AddAsync(w);
+            int affected = await db.SaveChangesAsync();
+
+             if (affected != 1)
+             {
+                 return null;
+             }
```

Comment:
Still an issue. Wasn't the if that should be moved, it was its body. The if statement should be aligned with the rest of the statements in the method body.

Reply:
i changes some settings so it should be easyer for me to spot the white space from now on, so this took so long for me to figure out how to notice